src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInDashboards.ts

```diff
@@ -2,12 +2,12 @@ import {
   ContentRecord,
   CubeName,
   DashboardState,
-  DataModel,
+  serializeDashboardState,
   deserializeDashboardState,
   MdxSelect,
   WidgetWithQueryState,
 } from "@activeviam/activeui-sdk-5.0";
-import { serializeDashboardState } from "@activeviam/activeui-sdk-5.1";
```

Comment:
Same question here, we decided [here](https://github.com/activeviam/activeui-migration/pull/52#discussion_r1039853461) to use `deserializeDashboardState` from 5.0 and `serializeDashboardState` from 5.1, what is the reason for this change ?